### PR TITLE
update license

### PR DESCRIPTION
**Pull request by Smokyish**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -3,7 +3,7 @@
                     GNU GENERAL PUBLIC LICENSE
                        Version 3, 29 June 2007
 
- Copyright (C) 2007 Free Software Foundation, Inc. <http://fsf.org/>
+ Copyright (C) Aragon Association
```
Comment (sohkai) [inline LICENSE]: Should we use aragon.org as the URL?

File: package.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
     "/test",
     "/truffle-config.js"
   ],
-  "author": "Aragon Institution MTU <contact@aragon.one>",
+  "author": "Aragon Association <contact@aragon.one>",
```
Comment (sohkai) [inline package.json]: Should we change this to `contact@aragon.org` or another aragon.org email?

Comment (luisivan) [reply package.json]: Indeed

File: LICENSE (hunk 1 above)
Comment (luisivan) [reply LICENSE]: +1

**Pull request merged**